src/test/java/com/patrickjones/services/TransferManagerTest.java

```diff
@@ -0,0 +1,32 @@
+package com.patrickjones.services;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.patrickjones.models.BancAccount;
+
+public class TransferManagerTest {
+
+	@Test
```

Comment:
`@Test` is an annotation that lets JUnit know to run that method as a unit test.